kubevirtci

```diff
@@ -96,11 +96,19 @@ function kubevirtci::install() {
 	${_kubectl} wait -n capk-system --for=condition=Available=true deployment/capk-controller-manager --timeout=10m
 }
 
+function kubevirtci::generate_kubeconfig() {
+        make clusterkubevirtadm-linux
+        bin/clusterkubevirtadm-linux-amd64 apply credentials --namespace e2e-test
+        bin/clusterkubevirtadm-linux-amd64 get kubeconfig --namespace=e2e-test --output-kubeconfig=kubeconfig-e2e
```

Comment:
Do we want the namespace to be hard coded? Maybe it can be the default, but that makes the function not usable for any other purpose than e2e test.

Reply:
I don't think that the namespace name is important, because the `clusterkubevirtadm` taking care to create it if it doesn't exists
Currently this function purpose is only for e2e-test, In case we would like to use it to another purpose in the future, we would the option to modify it